app/sessionStore.js

```diff
@@ -823,9 +823,15 @@ module.exports.runPreMigrations = (data) => {
   }
 
   if (data.lastAppVersion) {
+    let runHSTSCleanup = false
+    try { runHSTSCleanup = compareVersions(data.lastAppVersion, '0.22.00') < 1 } catch (e) {}
```

Comment:
this needs to be changed if this PR is going out in 0.23.x instead of 0.22.x

Reply:
update: since this is going out in 0.22.x release 2, this needs to be updated once we know the version number of 0.22.x release 1